src/main/java/org/wildfly/security/http/util/sso/DefaultSingleSignOnSession.java

```diff
@@ -160,6 +161,10 @@ public void put(SecurityIdentity identity) {
                             connection.getInputStream().close();
                         } catch (Exception cause) {
                             log.warnHttpMechSsoFailedLogoutParticipant(remoteURI.toString(), cause);
+                            // failed to logout participant, remove it from the list of participants
+                            try (SingleSignOn target = context.getSingleSignOnManagerManager().find(id)) {
```

Comment:
Can you also rename SingleSignOnContext.getSingleSignOnManagerManager() to getSingleSignOnManager()?  That was an unfortunate typo.